lib/util/cmcd_manager.js

```diff
@@ -410,6 +453,23 @@ shaka.util.CmcdManager = class {
       data.su = this.buffering_;
     }
 
+    if (
+      data.v === shaka.util.CmcdManager.Version.VERSION_2 &&
+      this.playerInterface_.isLive()
+    ) {
+      data.ltc = this.playerInterface_.getLiveLatency();
+    }
+
+    if (
+      data.v === shaka.util.CmcdManager.Version.VERSION_2 &&
+      !this.msdSent_ &&
+      this.playbackPlayingTime_ &&
+      this.playbackPlayTime_
+    ) {
+      data.msd = this.playbackPlayingTime_ - this.playbackPlayTime_;
+      this.msdSent_ = true;
+    }
+
```

Comment:
I think it's better to use a generic if statement for v2 and then manage each parameter inside. It would be clearer in the code which part is from v2 and which is not.